source/installer.py

```diff
@@ -197,12 +197,6 @@ def removeOldProgramFiles(destPath):
 			else:
 				os.remove(fn)
 
-	# #4235: mpr.dll is a Windows system dll accidentally included with
-	# earlier versions of NVDA. Its presence causes problems in Windows Vista.
-	fn = os.path.join(destPath, "mpr.dll")
-	if os.path.isfile(fn):
-		tryRemoveFile(fn)

```

Comment:
Do you have evidence that  not removing this file does not affect Windows 7 and up? True the comment said Windows Vista, but that is no doubt only because the comment was put there in the Windows Vista timeframe.

Reply:
If you're not sure, please revert this particular change.

Reply:
I decided to remove the change as you suggested. Indeed, I'm not 100% sure, and better safe than sorry.